### DISCUSSION TITLE
Keda with grafana

### DISCUSSION THREAD

**Discussion by carla-villar** [kedacore/keda#4685] [Q&A / Need Help]
### Proposal

Hi I have already deployed keda in kubernetes.
I need to add the following for grafana
 --set prometheus.metricServer.enabled=true

How we can do this

### Is this a feature you are interested in implementing yourself?

No

**Comment by JorTurFer** [comment]
Hi,
what do you mean? That's a helm parameter, you should deploy KEDA using helm to pass that parameter
